---
author:
title:
geometry: margin=.51in
papersize: letter
indent: true
documentclass: article
mainfont: "Crimson Text"
mainfontoptions: Ligatures=TeX,Scale=1.1
---

# Cover Sheet

\vspace{2cm}

## Better tools for DNA-based biodiversity surveys

\vspace{1cm}

## List of personel

\vspace{.5cm}

* **Post-Doctoral Associate:** Fran√ßois Michonneau, Whitney Laboratory for Marine
Bioscience.
[francois.michonneau@whitney.ufl.edu](mailto:francois.michonneau@whitney.ufl.edu)

* **PI:** Mark Martindale, Whitney Laboratory for Marine Bioscience. [mqmartin@whitney.ufl.edu](mailto:mqmartin@whitney.ufl.edu)

* **Co-PI:** Robert Guralnick, Florida Museum of Natural History. [robgur@gmail.com](mailto:robgur@gmail.com)

* **Co-PI:** Joseph Ryan, Whitney Laboratory for Marine
Bioscience. [joseph.ryan@whitney.ufl.edu](mailto:joseph.ryan@whitney.ufl.edu)

* **Co-PI:** Douglas Soltis, Florida Museum of Natural History, Department of Biology, Genetics Institute. [dsoltis@ufl.edu](mailto:dsoltis@ufl.edu)


----

**Project start date:** September 1st, 2016

**Amount of support requested:** $50,568

\pagebreak
\clearpage
\setcounter{page}{1}

### Introduction

High-Throughput sequencing (HTS) is poised to revolutionize how biodiversity is
studied. The parallel sequencing of all species found in a biological community
replaces the time-consuming steps of isolating and recognizing each species
found in an environmental sample. These approaches make it possible to estimate
the number of species occurring in the plankton, in the gut of fish (to infer
their diet), in marine sediments, and even in communities whose members cannot
be identified based on morphology [@Kartzinel2015a], [@Leray2015a]. Identifying
species in such communities based on morphology is time consuming, and often
imperfect: the identifications must be done on character-poor organisms (e.g.,
larvae in the plankton) or on animal fragments (e.g., a shrimp leg in a fish
gut). These sequencing technologies promise not only to estimate much more
accurately the number of species found in communities, but also (1) to identify
the species by comparing the DNA sequences to reference databases, and (2) to
provide genetic data that can be used to infer the phylogenetic relationships
among them. However, computing tools and software to deliver on these promises
are lacking, and the proposed research will help close this gap.

The study of how communities are assembled, how their composition is affected by
changes in the environment, or the answer to "How many species are there on
Earth?", has been limited by the absence of a rapid, inexpensive, and accurate
method to identify species. This is especially true for invertebrates that
include species-rich groups with limited taxonomic expertise. Molecular data has
transformed our understanding of marine biodiversity: how species are
distributed in space [@Schroth2002], how they interact [@Leray2015a], how they
are related to each others [@Dunn2008], and how to distinguish them
[@Knowlton2000]. Since the early 2000's, DNA barcoding (see Figure 1 and Box 1)
has been widely adopted to facilitate future identification, and uncover species
complexes (e.g., [@Michonneau2015-impatiens]). Almost 5 million DNA barcodes are
publicly available representing diverse taxonomic groups, from all latitudes,
covering both marine and terrestrial habitats. This dataset is the most
comprehensive for which the same gene has been sequenced, and is associated with
curated taxonomic information. To take advantage of this data, metabarcoding
(see Figure 1 and Box 1) targeting COI is emerging as the approach of choice to
characterize marine communities based on DNA [@Leray2015]. However, marine
biodiversity is still poorly characterized, and many taxa have not yet been
sequenced for COI. One of the challenges addressed by this proposed research is
to find the most likely taxon for a sequence given the species that have been
sequenced.

The realization that the phylogeny of locally co-occurring species could inform
on the environmental and biotic forces that determine which species occur in
communities has revitalized the field of community ecology
[@Webb2000],[@Webb2002]. However, a major limitation has been the difficulty of
acquiring a phylogeny for an arbitrary set of species. By generating DNA
sequences from the species occurring in the community, HTS and metabarcoding in
particular, are providing the data needed to infer the phylogenetic
relationships among these species. Here I propose to develop software that will
allow researchers to build phylogenies for the species in their communities
based on the sequences generated by metabarcoding.

Because of the prospects offered by metabarcoding, assembling high quality
reference databases of DNA barcodes is crucial. Additionally, to increase the
likelihood that the DNA sequences generated in metabarcoding studies will have
exact matches, most researchers build DNA barcode libraries in parallel to their
metabarcoding efforts. Few tools (and none that are open-source) exist to keep
the heterogeneous data generated during barcoding studies organized.

I propose to develop new tools and open-source software to increase the
scientific utility and value of the sequence data generated in metabarcoding
studies. I will test the performance and accuracy of these tools on simulated
datasets, and on metabarcoding datasets generated from marine communities. Over
the course of this project, I will work on three goals: (1) develop a
light-weight and expendable data management system for data generated in
barcoding surveys; (2) improve the assignment of taxonomic information to DNA
sequences generated during metabarcoding studies; (3) combine sequence data
generated by metabarcoding studies and taxonomic information to generate
community phylogenies.

This research is interdisciplinary and integrates biodiversity informatics,
evolutionary biology, ecology, and statistics. The results will help not only to
characterize and unravel biodiversity, but also to understand how communities
are assembled, how their members interact, and how the environment affect them.

![Comparison of workflows for barcoding and metabarcoding. With barcoding (A), tissues from specimens are used to isolate their DNA (gray strand). The barcoding gene (colored dots) is amplified by PCR, and a unique sequence for each specimen is obtained using Sanger sequencing. The sequences obtained can be used to build a reference library, where sequences are associated with a voucher specimen. With metabarcoding (B), DNA from an environmental sample (e.g., sea water containing planktonic larvae) is isolated. This solution contains DNA from multiple species, and the barcoding gene is amplified for the species. An High-Throughput Sequencing (HTS) platform (e.g., Illumina) is used to sequence all the amplified fragments of DNA (the amplicons). These sequences are analyzed bioinformatically: the sequences are matched against a reference database to identify the species contained in the environmental sample.](figures/drawing.pdf)


--------------------------------------------------------------------------------
**Box 1 --- Barcoding and COI** A portion of the cytochrome oxidase I (COI, a
mitochondrial gene) has emerged as a universal genetic marker to document
genetic diversity in animals. It is easy to amplify and the same primers can be
used across many animal groups. For most taxonomic group, COI has low
intra-specific but high inter-specific variation allowing species delineation
and identification.  A public database (BOLD, the Barcode Of Life Datasytem) has
been developed to store these DNA sequences alongside with voucher information
(in which natural history museum is the specimen from which the DNA comes from
held?), photography of specimens, and quality control on the sequence
data. Currently, this publicly available database holds almost 5 million
sequences from specimens collected worldwide, and from a large diversity of
taxonomic groups.

One of the strengths of DNA barcoding is that it allows the identification of
animals from parts, and from all stages of the life cycles. It becomes possible
to identify species found in fish guts to infer their diet, or to associate
larval forms to the adults (which butterfly is this caterpillar going to
metamorphose into? what is the adult stage of this larva found in the plankton?)

Among closely related species, COI sequences mostly vary at the third codon
position. Because of the redundancy of the genetic code, mutations on the third
codon affect less often the protein sequence than mutations at the other two
codons. Among more distantly related taxa, this excess of mutations on the third
codon may lead to convergent mutations. This may lead algorithms using sequence
similarity to assign sequences to the wrong taxa if no exact match is found.
However, these distantly related taxa will also include mutations on the other
codons that will lead to differences in their protein sequences. Because the
protein sequence will evolve slower than the nucleotide sequence, it will be
more accurate to infer the higher taxonomy from the protein sequence.

--------------------------------------------------------------------------------


### Research questions and datasets

The software developed for this research will be tested and used to analyze
metabarcoding datasets from the marine environment.

**Comparison of benthic and planktonic communities in contrasting environments
---** I will use two metabarcoding datasets generated on coral reefs. The first
was collected in the Line Islands, one of the most remote and uninhabited
archipelagoes of the tropical Pacific, between French Polynesia and Hawaii. With
collaborators from the Smithsonian Institution (Matthieu Leray and Nancy
Knowlton), we used metabarcoding to characterize the sessile fauna colonizing
the skeletons of dead corals, as well as the planktonic communities. We
assembled a similar dataset from Bocas del Toro, a lagunar system heavily
impacted by human activities on the Caribbean coast of Panama. In both
locations, we also supplemented these data by assembling a reference DNA library
for the major groups composing the fauna. I will use the tools developed in this
project to assess how many species are found in each habitat (sessile
vs. plankton), how many species can be found in both habitats, how closely
related species found in each habitat are, and how these numbers differ between
the Line Islands and Panama.

**Temporal dynamics of community relatedness in the plankton ---** Since
September 2015, I have been sampling plankton biweekly in the Matanzas River
near the Whitney Laboratory. I will use the tools developed in this research to
understand how the taxonomic and phylogenetic composition of the planktonic
communities has changed through the year, and which environmental factors
influence the groups present in the plankton. In parallel to collect samples for
metabarcoding, I have photographed and individually sequenced larvae found in
the plankton to assemble a reference DNA library. This dataset will allow me to
ground truth the results I obtain from the metabarcoding.


### Approach and methods



**Goal 1 -- a data management system for DNA barcoding projects.** Assembling a
DNA barcoding library requires to keep track of heterogeneous data: where the
specimen was collected, in which natural history museum it is deposited, its
taxonomic information, photographs of the specimen, DNA extractions, and DNA
sequence data. BOLD ([\mbox{http://boldsystems.org}](http://boldsystems.org))
provides an online platform to organize most of this data, and facilitates the
direct submission of DNA sequences to Genbank
([http://www.ncbi.nlm.nih.gov/genbank/](http://www.ncbi.nlm.nih.gov/genbank/)).
However, there is no software that allows researchers to manage, visualize, and
prepare their data locally before uploading to BOLD.

I propose to develop a data management system that will allow researchers to
easily create, organize, and track all the digital artifacts and they generate
over the course of assembling a DNA barcoding library. By facilitating the
organization of the data, this software will improve the quality of the data by
ensuring that all the digital artifacts are properly linked.

I want this system to be easy to use and manage for non technical users. I will
develop it as an R package that will provide a programmatic interface to the
data, and with a companion web-application written using Shiny
(http://shiny.rstudio.com/) for the front-end. R is widely used by evolutionary
biologists and ecologists. By developing this data management system in R,
researchers will be able to contribute to it, and expand it to fit their own
needs. The web-based front-end will allow non-technical users to explore the
data. The data will be stored using the Darwin Core standards for biodiversity
data, and the software will provide functionalities to export the data to BOLD.

To make it easy to install, deploy, and backup, the data used by this management
system will be stored in a SQLite database and a transparent file system. To
facilitate deployment, I will also maintain a Docker image that will provide all
the software components needed. I have started the development of a proof of
concept of this system available that is available at
[http://104.131.113.146/flpk-shiny/](http://104.131.113.146/flpk-shiny/). The
code for the source for the R package is available at
[https://github.com/fmichonneau/labmanager](https://github.com/fmichonneau/labmanager)
and the front-end at
[https://github.com/fmichonneau/flpk-shiny](https://github.com/fmichonneau/flpk-shiny).

**Goal 2 -- improving DNA sequence classification for COI metabarcode.** Current
software used to assign taxon names to DNA sequences generated with HTS are
ill-suited for animal communities. These software were originally designed to
analyze sequences originating from microbial communities
[@Bazinet2012]. Metabarcoding studies on bacteria typically target the genetic
marker 16S. It is known to provide good resolution at low taxonomic level and is
easy to amplify across a large diversity of species. Several databases of 16S
sequences for bacteria have been assembled. Because 16S does not encode for a
protein, it evolves differently from COI, and the algorithms used to match the
16S sequences to the reference database do not take advantage of the
protein-coding nature of COI.

Current metabarcoding methods use only DNA sequences to identify exact or close
matches in a reference database. I propose to modify the current workflow used
in DNA sequence classification to also include a step where the DNA sequences
get automatically translated into their protein sequence. By being less variable
than DNA sequences, protein sequences will increase the probability of finding a
match in public databases, and narrow down the taxonomic scope to which the DNA
sequence can be matched. This approach will complement existing software that
provide adequate results when an exact or close match exist in the reference
database. However, because many marine taxa are incompletely sampled, the
probability for an exact or close match is generally low.

On a dataset of COI sequences for planktonic larvae of marine invertebrates
collected in Florida (a relatively well studied location for its marine fauna),
less than 25% have close or exact matches in the BOLD database. This number is
expected to be even lower when sequences from environmental sample will be
analyzed. When using the software SAP, a popular sequence classifier routinely
used in metabarcoding studies, only 33% of these sequences could be assigned at
the phylum level, and 18% at the species level (using GenBank). These low levels
of assignment stem from the lack of significant homolog matches when comparing
input DNA sequences and sequences available in public databases. In other words,
sequences available in reference databases are too divergent from the sample
sequences (closely related taxa have not been sequenced). However, based on the
protein sequences, 76% were assigned the correct phylum.

The software SAP [@Munch2008],[@Munch2008-NJ] is routinely used in metabarcoding
studies because it uses two complimentary approaches when attempting to find a
match in the reference database (while most other software use only one). It
also provides a confidence score for this assignment (something few software
do). First, the sequence to be matched is used to look for homolog sequences
based on sequence similarity (a BLAST search). The reference sequences that
match best and the sample sequence are aligned. This alignment, and bootstrap
replicates of it, are used to estimate phylogenetic trees. From this set of
trees, the frequency with which the sequence to assign clusters with the same
taxa gives a confidence value for this assignment. By using the higher taxonomy
of the reference sequences similar confidence values are given at the genus,
family, order, class and phylum level.

I will develop a new software that will modify the approach taken by
SAP. Instead of looking for homolog sequences based only on the nucleotide
sequences, I will in addition include homologs based on their respective protein
sequences. This approach will increase the likelihood of finding matches in the
reference database, and will allow the assignment to a higher-level taxon when
no exact match is found. I will develop this software in Python as it is a
popular language for bioinformatics and is well-suited for this type of
application.

To test the accuracy of this approach, I will use sequences from known species
and use the software developed to attempt to identify them. To simulate the
effect of trying to identify a species not represented in the reference
database, I will ignore all the hits for this particular species and use the
next best matches. I will repeat this operation at increasingly higher taxonomic
levels (removing the genus, the family, the class, the order) to determine how
the software behaves when the sequence to be classified is taxonomically further
away from taxa available in the reference database. Because the results are
likely dependent on how many representatives of the group are included in the
reference database, the test dataset will be assembled to include representative
from diverse taxa.

<!--Another area of improvement for software aimed at analyzing metabarcoding data
is during the assessment of the quality of the sequences. It is common to
translate DNA sequences into their protein sequence, and remove sequences that
include stop codons. Those could arise from sequencing error, or from the
amplification of pseudo-genes. This quality control step is done using the
standard genetic code. However, across the diversity of animal phyla that can be
found in metabarcoding studies, especially in the marine realm, alternative
genetic codes are encountered (e.g., echinoderms, cnidarians,
platyhelminthes). These codes translate a codon interpreted as a stop codon in
the standard code as a valid amino-acid (UGA). Thus, DNA sequences encoding for
functional COI proteins get discarded, removing systematically some taxa from
the analyses. My software will include a more careful quality control step that
will account for the presence of these alternative genetic codes.-->


**Goal 3 -- obtaining community phylogenies from metabarcoding data.** A recent
exciting research development in ecology is the realization that understanding
how closely related species that are interacting in a community can shed light
on the ecological processes at play. For instance, if species occurring in a
given environment are more closely related than would be expected by chance, it
can be hypothesized that characteristics of the environment select for a trait
that these species have (ecological filtering). On the other hand, if species
are more evenly spaced in the phylogeny than one would expect by chance, it can
be hypothesized that competitive exclusion precludes related species to
co-exist. Theory, mathematical models, and software to test such hypotheses have
been developed. However, advances in community phylogenetics have been hampered
by the difficulty of obtaining a phylogeny for an arbitrary set of species.

The Open Tree of Life project
([https://tree.opentreeoflife.org](https://tree.opentreeoflife.org)) provides a
topology for all named organisms. This tree is assembled by combining taxonomic
information and information from published phylogenies. However, this global
tree does not include branch lengths, and therefore cannot be used in
statistical analyses as the ones required by community phylogenetics analyses.

The DNA barcoding gene used for animals (COI) evolves fast enough that it can
reconstruct relationship among closely related species with good
accuracy. However, its rate of evolution does not allow the accurate
reconstruction of relationships among more distantly related species.

I propose to combine information from the Open Tree of Life project and the COI
data obtained through metabarcoding to estimate the phylogeny of species
included in the meta-barcoded communities. To do so, I will use the taxon
assignments inferred from the COI sequences (using the software developed for
Goal 2) to retrieve the topology for these species from the Open Tree of
Life. This topology will be used as a constraint when estimating the phylogeny
from the COI data. Hence, I will take advantage of what COI does best (resolving
relationship among closely related species), and compensate its limitation by
using the topology provided by the Open Tree of Life project. To validate the
accuracy of this approach, I will compare phylogenies generated by this method
with published phylogenies reconstructed based on multi-gene DNA alignments.

I will develop this software using R as it is a very popular language to work
with phylogenies, and most methods for community phylogenetics have been
implemented with it. Additionally, I have already written an R package to
interface with the Open Tree of Life web API to facilitate the retrieval of the
topology for an arbitrary set of taxa [@Michonneau2015].


### Intellectual merit

Metabarcoding is transforming how biodiversity is characterized, but the tools
currently available are inadequate. They only use a small part of the scientific
potential these data have. By providing software specifically designed to
analyze metabarcoding data from protein-coding genes, the proposed research will
improve our understanding of biodiversity, and the results can be used to
address long-standing questions in ecology pertaining to natural history,
community assembly, and species interactions.

By focusing on marine communities, this project will have to deal with diverse
taxonomic groups, some of which have limited information available in public
databases. These constraints will make the software developed robust to other
systems that focus on communities composed of more closely related taxonomic
groups (e.g., soil arthropods, freshwater vertebrates) or on groups that rely on
other protein coding genes for barcoding (e.g., plants).

The intellectual environment the UF Informatics Institute provides by bringing
together postdoctoral fellows from diverse backgrounds will be exciting, and
will facilitate working on solving complex scientific problems.


### Broader Impacts

*Scientific community outreach ---* All the source code generated by this
project will be open source, and development will be open to collaboration as it
will be hosted on GitHub
([https://github.com/fmichonneau](https://github.com/fmichonneau)). The software
will be comprehensively documented and include tutorials to guide first time
users. Additionally, I will also solicit iDigBio's (https://idigbio.org) support
to organize a workshop aimed at defining best archival practices and the type of
metadata that should be associated with environmental samples.

*Pedagogical outreach ---* I will teach workshops that demonstrate how to use
the software developed over the course of this project. As a certified Data
Carpentry and Software Carpentry instructors (see Biosketch), I will help
organize such workshops on campus. These provide a venue to increase
computational literacy on campus, advertise available resources, and ultimately
grow the community of users who collaborate on solving computing problems.

*Public and K-12 outreach ---* Marine organisms provide unique opportunities to
teach about adaptations and spark the sense of wonder for the natural world. I
plan to use my research to develop existing and new learning activities for
high-school students and for the general public.

### Timeline

\includegraphics{figures/timetable}

\pagebreak



# Project Budget

A budget of $50,568 is requested. This include a year of salary for the
postdoctoral associate ($42,000) with a 20.4% fringe benefit rate as listed
here: http://research.ufl.edu/faculty-and-staff/proposal-development-submission/budgeting-information/budget-preparation-information.html

This will fund year 1 of the research proposed here. The PI (Mark Martindale)
will provide salary for year 2 (see department coversheet, and letter of support
included in Appendix 2).

\pagebreak

# References





<!-- First, the sequences get translated according to the candidate codon -->
<!-- tables given the taxa targeted by the analysis. Codon tables that lead to stop -->
<!-- codons are not retained. The protein sequence is then matched against a database -->
<!-- of representative sequences for COI across of the tree of life. This steps -->
<!-- allows the assignment of this unknown sequence to a phylum. If the inferred -->
<!-- phylum is compatible with one of genetic codes inferred, the phylum and the -->
<!-- genetic codes are retained. -->


<!-- Slow process because sequences have to be queried over all of GenBank which is -->
<!-- huge and not necessarily relevant. Improve speed by providing a set of sequences -->
<!-- from organisms most likely to occur in the sample, and for those that don't -->
<!-- match, then do a BLAST search, possibly limiting to a phylum based on the -->
<!-- sequenc -->

<!-- Paragraph on utility and accuracy of barcoding for animals. When dataset is -->
<!-- looked under increased scrutinity, usually very low percentage of -->
<!-- missassignment. Here givne the goal, it wouldn't be too big of a deal. Narrowing -->
<!-- an unidentified sequence to a few species possibilities would be better than -->
<!-- what is currently possible. -->


<!--  Species description of marine -->
<!-- organisms has typically been done on preserved specimens whose coloration has -->
<!-- been washed off by preservatives used for fixation, and with little information -->
<!-- about their live appearance or their habits. The incorporation of molecular data -->
<!-- into taxonomy has unraveled high levels of unrecognized diversity with many -->
<!-- cryptic species. Hence, species thought to be widespread appeared in the light -->
<!-- of molecular data to form mosaics of species complexes. Under scrutiny, these -->
<!-- newly recognized species differed in their coloration and habitat, corroborating -->
<!-- the results from molecular data. -->


<!-- DNA barcoding has changed how marine biodiversity is studied, with large, -->
<!-- all-taxa inventories with vouchers stored in museums, images of the specimens, -->
<!-- and sequence data from COI. Besides improving the documentation of the number of -->
<!-- species, barcoding data has also helped unravel the complex life cycles of many -->
<!-- organisms, connecting the larval stage to the adult, or been used to assign -->
<!-- species to the animal parts found in the stomachs of fish to reconstruct their -->
<!-- diet. Biodiversity science is undergoing another transformation with the advent -->
<!-- of metabarcoding. Instead of sequencing individuals, metabarcoding takes -->
<!-- advantage of the massive parellel sequencing that next generation sequencing -->
<!-- platforms provide. Contrary to metagenomics, metabarcoding relies on amplicon -->
<!-- sequencing where the DNA from an environmental sample is extracted before the -->
<!-- genetic marker of interest is amplified and sequenced. -->
